wp-content/themes/access/src/js/modules/office-map.js

```diff
@@ -114,21 +116,30 @@ class OfficeMap {
       this.updateList().updateUrl();
     });
 
-    // Bias the SearchBox results towards current map's viewport when the map
-    // bounds change.
+    // Bias the Autocomplete's results towards current map's viewport when
+    // the map bounds change.
     this._map.addListener('bounds_changed', _.debounce(() => {
-      this._searchBox.setBounds(this._map.getBounds());
+      this._Autocomplete.setBounds(this._map.getBounds());
     }, 100));
 
+    // Avoid paying for data that you don't need by restricting the set of
+    // place fields that are returned to just the geometry components.
+    this._Autocomplete.setFields(['geometry']);
```

Comment:
This is supposedly where we can save some $$ on billing by specifying that we only need this data from the autocomplete.getPlace() method.